File: reni-backend/src/main/java/pe/gob/mimp/reni/dto/request/EstructuraRegistrarParametroRequest.java
package pe.gob.mimp.reni.dto.request;

import java.io.Serializable;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class EstructuraRegistrarParametroRequest implements Serializable {
	private static final long serialVersionUID = 1L;

	private String nomCampoExcel;
	private Integer ordenCampoExcel;
	private String nomTablaBd;
	private String nomCampoBd;
	private Integer campoEsFk;
	private String campoIdmaestra;
	private String descripcion;
}

File: reni-frontend/src/main/reni-frontend/src/app/modules/intranet/dto/request/estructura-registrar-parametro.request.ts
export class EstructuraRegistrarParametroRequest {
    nomCampoExcel: string;
    ordenCampoExcel: number;
    nomTablaBd: string;
    nomCampoBd: string;
    campoEsFk: number;
    campoIdmaestra: string;
    descripcion: string;

    error: any[] = [];
    errorTablaBDExiste: boolean;
    errorCampoBDExiste: boolean;

}